include/rbs/rbs_location.h

```diff
@@ -5,7 +5,27 @@
 
 #include "rbs/util/rbs_constant_pool.h"
 #include "rbs/util/rbs_allocator.h"
-#include "rbs/rbs_location_internals.h"
+
+typedef struct {
+  int start;
+  int end;
+} rbs_loc_range;
+
+typedef struct {
+  rbs_constant_id_t name;
+  rbs_loc_range rg;
+} rbs_loc_entry;
+
+typedef unsigned int rbs_loc_entry_bitmap;
+
+// The flexible array always allocates, but it's okay.
+// This struct is not allocated when the `rbs_loc` doesn't have children.
```

Comment:
Aside, but this is a weird comment. There is no flexible array member here (that would be `rbs_loc_entry entries[]`), and there's no allocation there, either.

Reply:
FWIW this is the PR introduced the usage: https://github.com/ruby/rbs/pull/1786